content/docs/user-guide/how-to/best-practices.md

```diff
@@ -13,6 +13,11 @@ is required.**
   [revisions](https://git-scm.com/docs/revisions) of source code and versions of
   data files, when the data cannot fit into a traditional repository?
 
+### Experiments
+
+- How do you document progress of training different types of models on your
+  data files in the same project?
```

Comment:
If am not wrong this material was previously named as Collaboration issues. I think this isn't required here. But if we want to keep this it can be under be a document under User guide with something like 'Why DVC?' but to not required to be honest.

Reply:
I have updated this doc with some checkboxes from #72 but haven't deleted this material yet.

Reply:
I don't think it's from **Collaboration Issues** but it doesn't matter. Maybe leave it for now as a FAQ section at the bottom of this guide, but several of these questions could be turned into practices for sure e.g. "How do you document progress of training different types of models?"

Reply:
But my question is what is your plan for this doc, after having read these questions, and issue #72? BTW you may want to also search "practice", "best practice", "recommended way", and similar phrases in the Discord channel to look for other common questions that users have.

Reply:
Yeah I thought this doc should address checkboxes in #72 . But now realised they are separate issues and best practices is a different doc. I have updated the doc and now its similar to QnA. The answers reflect what is the best way to achieve the solution to the question. Thoughts on the format?

Reply:
It's an interesting format but there's some confusion. Maybe I shouldn't have combined both issues in this PR, sorry. And yes, they are separate issues (start How-to section #899 and write a Best Practices guide #72) but closely related because you can't start a How-to section without having a few how-to guides, and I think The Best Practices guide belongs in this section (along with the other guide I already moved into /how-to when creating this this PR). See?

Reply:
> best practices is a different doc

Summarizing: How-to is not a doc., but a section. Best Practices is the first new doc in that section.

Reply:
But my questions on the new Best Practices doc remain, please address them directly: what is your plan for this doc? Which best practices have you researched (or need help with)? Thanks

Reply:
I think Best Practices doc looks good as first doc in How-to section. I have added a point on how to track outs by Git instead of DVC(--out-no-cache in `run`). I think another one can be best way to import DVC tracked files from other DVC repos(import stage). Apart from these i'm searching for more.

Reply:
Asking again:
- What is your plan for this doc? I'm still not seeing a clear direction.
- Which best practices have you researched (or need help with)?

Reply:
in my view this doc will be better under user-guide directly. It should be single doc. There will be some best practices (like **Managing and sharing large data files**) which already have examples in other docs so we can explain the best practice to follow and then link it to the already existing example. single doc should be good and it can be more generalized. If a best practice is big enough then it can be a separate doc directly under how-to section.

Reply:
OK @imhardikj, in that case please extract the best practices guide into a separate draft PR (use "WIP" in the title for now). That PR should aim to completely close #72.

> If a best practice is big enough then it can be a separate doc directly under how-to section.

That didn't make much sense to me but OK, not something we have to decide right now.

Reply:
Have created the new PR #1748. Answered https://github.com/iterative/dvc.org/pull/1705#issuecomment-683352601 here only.
This PR can be merged after moving existing how-to's to this section. Apart from `Updating Tracked Files`, `running on windows` can also be moved here. WDYT
But I think it would be better to add more docs in this sub section  based on issues in #899. This way this PR will be substantial and have much more content. Thanks.

Reply:
> This PR can be merged after moving existing how-to's to this section

Oh no, that's just the beginng haha. We need to add some new content 🙂 pick something from #899

> running on windows can also be moved here. WDYT

Nah, that content is more like tips. Not how-to content (goal oriented). Again, please review this so it's extra clear what a how to is: https://documentation.divio.com/how-to-guides/#how-to

You're right that one doc as well as others like **Related Technologies** may not really belong in _User Guide_ though, but that's a completely separate problem, let's not further complicate this task for now 🙂

Reply:
> Answered https://github.com/iterative/dvc.org/pull/1705#issuecomment-683352601 here only.

What was the answer? 🙂

Reply:
> What was the answer

@jorgeorpinel  I meant that it would be better to add more docs based on issues in #899 in this PR. That's the plan for this PR.

I have added a how-to doc based on #625 .